src/builder/specification.md

```diff
@@ -0,0 +1,349 @@
+- `gasLimit`: `QUANTITY`, 64 Bits
+- `gasUsed`: `QUANTITY`, 64 Bits
+- `timestamp`: `QUANTITY`, 64 Bits
+- `extraData`: `DATA`, 0 to 32 Bytes
+- `baseFeePerGas`: `QUANTITY`, 256 Bits
+- `blockHash`: `DATA`, 32 Bytes
+- `transactionsRoot`: `DATA`, 32 Bytes
+
+### `BlindBeaconBlockV1`
+- `slot`: `QUANTITY`, 64 Bits
+- `proposerIndex`: `QUANTITY`, 64 Bits
+- `parentRoot`: `DATA`, 32 Bytes
+- `stateRoot`: `DATA`, 32 Bytes
+- `body`: `object`, [`BlindBeaconBlockBodyV1`](#blindbeaconblockbodyv1)
+
+### `BlindBeaconBlockBodyV1`
+- `randaoReveal`: `DATA`, 96 Bytes
+- `eth1Data`: `object`, [`Eth1DataV1`](#eth1datav1)
+- `graffiti`: `DATA`, 32 Bytes
+- `proposerSlashings`: `object`, [`ProposerSlashingV1`](#proposerslashingv1)
+- `attesterSlashings`: `object`, [`AttesterSlashingV1`](#attesterslashingv1)
+- `attestations`: `Array`, [`AttestationV1`](#attestationv1)
+- `deposits`: `Array`, [`DespositV1`](#depositv1)
+- `voluntaryExits`: `Array`, [`SignedVoluntaryExitV1`](#signedvoluntaryexitv1)
+- `syncAggregate`: `object`, [`SyncAggregateV1`](#syncaggregatev1)
+- `executionPayload`: `object`, [`ExecutionPayloadHeaderV1`](#executionpayloadheaderv1)
+
+### `Eth1DataV1`
+- `depositRoot`: `DATA`, 32 Bytes
+- `depositCount`: `QUANTITY`, 64 Bits
+- `blockHash`: `DATA`, 32 Bytes
+
+### `ProposerSlashingV1`
+- `signedHeader1`: `object`, [`SignedBeaconBlockHeaderV1`](#signedbeaconblockheaderv1)
+- `signedHeader2`: `object`, [`SignedBeaconBlockHeaderV1`](#signedbeaconblockheaderv1)
+
+### `BeaconBlockHeaderV1`
+- `slot`: `QUANTITY`, 64 Bits
+- `proposerIndex`: `QUANTITY`, 64 Bits
+- `parentRoot`: `DATA`, 32 Bytes
+- `stateRoot`: `DATA`, 32 Bytes
+- `bodyRoot`: `DATA`, 32 Bytes
+
+### `AttesterSlashingV1`
+- `attestation1`: `object`, [`IndexedAttestationV1`](#indexedattestationv1)
+- `attestation2`: `object`, [`IndexedAttestationV1`](#indexedattestationv1)
+
+### `IndexedAttestationV1`
+- `attestingIndices`: `Array`, `QUANTITY`, 64 Bits
+- `data`: `object`, [`AttestationDataV1`](#attestationdatav1)
+- `signature`: `DATA`, 96 Bytes
+
+### `AttestationDataV1`
+- `slot`: `QUANTITY`, 64 Bits
+- `index`: `QUANTITY`, 64 Bits
+- `beaconBlockRoot`, `DATA`, 32 Bytes
+- `source`: `object`, [`CheckpointV1`](#checkpointv1)
+- `target`: `object`, [`CheckpointV1`](#checkpointv1)
+
+### `CheckpointV1`
+- `epoch`: `QUANTITY`, 64 Bits
+- `root`: `DATA`, 32 Bytes
+
+### `AttestationV1`
+- `aggregationBits`: `DATA`, 0 to 256 Bytes
+- `data`: `object`, [`AttestationDataV1`](#attestationdatav1)
+- `signature`: `DATA`, 96 Bytes
+
+### `DepositV1`
+- `proof`: `Array`, 32 Bytes
+- `data`: `object`, [`DepositDataV1`](#depositdatav1)
+
+### `DepositDataV1`
+- `pubkey`: `DATA`, 48 Bytes
+- `withdrawalCredentials`: `DATA`, 32 Bytes
+- `amount`: `QUANTITY`, 64 Bits
+- `signature`: `DATA`, 96 Bytes
+
+### `VoluntaryExitV1`
+- `epoch`: `QUANTITY`, 64 Bits
+- `ValidatorIndex`: `QUANTITY`, 64 Bits
+
+### `SyncAggregateV1`
+- `syncCommitteeBits`: `DATA`, 0 to 64 Bytes
```

Comment:
sync committee bits are fixed to 512 bits (`Bitvector[SYNC_COMMITTEE_SIZE]`). Shouldn't it just be? 👇🏼 
```suggestion
- `syncCommitteeBits`: `DATA`, 64 Bytes
```

Reply:
Ah good catch, thanks.